routes/topics.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const router = express.Router();
+
+const {getAllTopics} = require('../controllers/topics');
+
+router.get('/', getAllTopics);
```

Comment:
Please take another look at routing! At the moment, you have it set up, but aren't utilising it correctly. You're currently saying that any request that comes into your app should respond with allTopics.

Reply:
One option would be to leave the routing to one side until you're a bit further along in the project. This would mean you could do something like app.get('/topics', getAllTopics) and build up the rest of your endpoints accordingly!